app/javascript/app/pages/climate-policies/instruments/instruments-styles.scss

```diff
@@ -0,0 +1,99 @@
+@import '~styles/ui-extends.scss';
+@import '~styles/layout.scss';
+
+.page {
+  margin-bottom: 50px;
+}
+
+.pageLayout {
+  @include row();
+}
+
+.section {
+  @include columns();
+
+  padding-left: $gutter-padding / 2;
+  margin: {
+    top: $gutter-padding;
+    bottom: $gutter-padding * 3;
+  }
+}
+
+.titleContainer {
+  display: flex;
+  justify-content: center;
+  background-color: $link-water;
+  border-bottom: 1px solid rgba($jakarta, 0.3);
+  color: $downriver;
+}
+
+.title {
+  padding: 30px;
+  text-transform: uppercase;
+  font-size: $font-size-large;
+  font-weight: bold;
+}
+
+.cell {
+  padding: 20px;
+  color: $downriver;
+  line-height: 1.5;
+}
+
+.nameColumn {
+  text-align: right;
+  font-weight: bold;
+}
+
+.table {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  border-bottom: 1px solid rgba($jakarta, 0.2);
+
+  > * {
+    flex: 0 1 33%;
+  }
+}
+
+.icon {
+  margin-left: 10px;
+}
+
+.link {
+  text-decoration: underline;
+  color: $dark-blue;
+}
+
+.document {
+  display: flex;
+  align-items: center;
+  font-size: $font-size;
+  cursor: pointer;
+}
+
+.accordionTitle {
+  justify-content: center;
+  padding-right: 0;
+}
+
+.accordionHeader {
+  background-color: rgba($link-water, 0.2);
+  border-bottom: 1px solid rgba($jakarta, 0.2);
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba($link-water, 0.2);
+  border-bottom: 1px solid rgba($jakarta, 0.2);
+  padding: 20px;
+}
+
+.date {
+  color: $elephant-blue;
+  opacity: 0.4;
```

Comment:
Are we using opacity instead of a different color to match the design colors? Maybe we could use lighten(color, 60%) to avoid transparency issues.

Reply:
good point. I used `rgba` to make it a little transparent.